src/components/home.jsx

```
@@ -1,14 +1,15 @@
 import React, { Component } from 'react';
 import Grid from './grid';
+/* eslint react/prefer-stateless-function: 0 */
```

don't suppress this error

---

you can just make Popup a presentation component, checkout presentational v container components.

i.e.
```
const Component = props => (
 <render material goes here>
);


export default Component;
```